src/server/benchpress_server.ml

```diff
@@ -1746,9 +1746,9 @@ let handle_file self : unit =
       Log.debug (fun k -> k "get-file: `%s`" file);
       let bytes =
         if file = "prelude" then
-          H.Byte_stream.of_string Builtin_config.config (* magic file! *)
+          H.IO.Input.of_string Builtin_config.config (* magic file! *)
```

Comment:
I think this needs an additional lower bound on  tiny_httpd ! (I think 0.14 would do it).

Reply:
ha!, didn't see the last change

Reply:
I think these changes are specific to the `tiny_httpd` version 0.17.0. It seems there's no longer `Tiny_httpd_stream` so I used `Tiny_httpd.IO.Input` instead.